### PR TITLE
use citus in travis tests

### PR DESCRIPTION
**Pull request by calellowitz**
##### SUMMARY

Kicking off an attempt to get travis tests to run on citus, but want to start the build sooner rather than later. I suspect there is a bit more work to be done, but this is a good starting point.

### CODE REVIEW

**Review by czue — APPROVED** [review]
:pray:

File: custom/icds_reports/tests/test_aggregation_script.py [hunk 1]
```diff
@@ -553,7 +555,7 @@ def setUpClass(cls):
         sup2.metadata = {"is_test_location": "test"}
         sup2.save()
 
-        cls.helper = LocationAggregationHelper()
+        cls.helper = LocationAggregationDistributedHelper()
```
Comment (czue) [inline custom/icds_reports/tests/test_aggregation_script.py]: can we set this conditionally based on `settings.ICDS_USE_CITUS` or similar?

**Review by czue — COMMENTED** [review]
Looks like build still failing (I've seen this locally too).

```
======================================================================
ERROR: custom.icds_reports.tests.test_aggregation_script:LocationAggregationTest.test_agg
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/test_aggregation_script.py", line 588, in test_agg
    self.helper.aggregate(cursor)
  File "/mnt/commcare-hq-ro/custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py", line 99, in aggregate
    self.assert_no_awc_missing_from_new_table(cursor)
  File "/mnt/commcare-hq-ro/custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py", line 150, in assert_no_awc_missing_from_new_table
    raise LocationRemovedException(str(num_locations_missing))
custom.icds_reports.exceptions.LocationRemovedException: 55
```

File: custom/icds_reports/tests/test_aggregation_script.py (hunk 1 above)
Comment (calellowitz) [reply custom/icds_reports/tests/test_aggregation_script.py]: We can, but the distributed version is guaranteed to work on a non-citus db, so it does not buy us much (I am a little surprised the original implementation did not use that one already)

Comment (czue) [reply custom/icds_reports/tests/test_aggregation_script.py]: Oh neat, I didn't know that.

**Comment by emord** [comment]
passes after https://github.com/dimagi/commcare-hq/pull/25453/commits/126863191e0335e8c680c0936a1d48fdb120b6fc 

Those tests are cleaning up the awc_location tables for the dashboard before re-aggregating them, but Citus uses `awc_location_local` as the source of truth since its not a distributed table. The only reason non-Citus didn't do that was because I wrote it first using `awc_location` (not realizing that that wasn't going to work for Citus)

**Comment by czue** [comment]
> passes after 1268631

:tada: :clap: 

Any reason not to merge this?

**Comment by emord** [comment]
Didn't self-merge because I made a non-trivial change at the end. Taking your comment as approval.

**Pull request merged**